cmd/call-api-endpoint/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/edulinq/autograder/internal/api"
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/log"
+)
+
+var args struct {
+	config.ConfigArgs
+	cmd.CommonOptions
+
+	Endpoint   string   `help:"Endpoint of the desired API." arg:""`
+	Parameters []string `help:"Parameter for the endpoint in the format 'key:value', where the first colon separates the key and value (e.g., 'target-email:bob@test.edulinq.org'). Additional colons are allowed in the value." arg:"" optional:""`
```

Comment:
Make this text more terse (help prompts and man pages are typically pretty terse).
Everything past "'key:value'" seems kind of useless.
I could see giving a small example, but use shorter strings for it.